Library/Homebrew/extend/os/linux/development_tools.rb

```diff
@@ -19,5 +19,10 @@ def locate(tool)
     def default_compiler
       :gcc
     end
+
+    def build_system_info
+      glibc_version = Formula["glibc"].any_installed_version&.version || OS::Linux::Glibc.system_version
```

Comment:
Definitely don't want `pkg_version` here? I could see that being useful for debugging.

Reply:
`pkg_version` works for me. I'm primarily concerned with the value in our bottles, where brewed `glibc` is not installed and `glibc_version = OS::Linux::Glibc.system_version`.